surveys/views.py

```diff
@@ -52,6 +52,12 @@ def form_valid(self, form):
 
         return HttpResponseRedirect(self.get_success_url())
 
+    def get_success_url(self):
+        form_entry_id = self.object.formentry_ptr_id
+        success_url = reverse_lazy('fobi.edit_form_entry', kwargs={'form_entry_id': form_entry_id})
```

Comment:
Any particular reason to use `reverse_lazy` instead of `reverse` here? Seems like both work, but I can't quite tell if there's an important difference in this context.

Reply:
It would work either way here! Do you know an advantage to using plain `reverse`?

At first I tried defining `success_url` at the top of the class outside a method with `reverse`, and ran into [this issue](https://stackoverflow.com/questions/31275574/reverse-for-success-url-on-django-class-based-view-complain-about-circular-impor) which was solved by using `reverse_lazy`. When I switched to modifying `get_success_url` I didn't see a reason to change it back.

Reply:
Gotcha! No advantage I can think of eithe rway, I was just curious if there was a reason that the URLconf wouldn't be loaded at this point (as is the problem with in `get_success_url` that requires using `reverse_lazy` instead of `reverse`). I could see an argument for using `reverse` since it won't mislead another developer into thinking that there's a particular importance to `reverse_lazy` here, but both should work fine.